pdfrw/objects/pdfstring.py

```diff
@@ -316,6 +329,11 @@ class PdfString(str):
     indirect = False
 
 
+    # The byte order mark, in both bytes and unicode form
+
+    bytes_bom = codecs.BOM_UTF16_BE
+    uni_bom = bytes_bom.decode('latin-1')
```

Comment:
I still think that this is not what we should check at the beginning of a unicode string. `uni_bom` here contains the code points called:

- LATIN SMALL LETTER THORN
- LATIN SMALL LETTER Y WITH DIAERESIS

The BOM is a single code point, `U+FEFF`. You can make it with e.g. `bytes_bom.decode('utf-16-be')`